pkg/internal/app/wopiinfo.go

```diff
@@ -101,10 +114,23 @@ func CheckFileInfo(app *demoApp, w http.ResponseWriter, r *http.Request) {
 
 	jsonFileInfo, err := json.Marshal(fileInfo)
 	if err != nil {
+		app.Logger.Error().
+			Err(err).
+			Str("FileReference", wopiContext.FileReference.String()).
+			Str("ViewMode", wopiContext.ViewMode.String()).
+			Str("Requester", wopiContext.User.GetId().String()).
+			Msg("CheckFileInfo: failed to marshal fileinfo")
 		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
 		return
 	}
 
+	app.Logger.Debug().
+		Str("FileReference", wopiContext.FileReference.String()).
+		Str("ViewMode", wopiContext.ViewMode.String()).
+		Str("Requester", wopiContext.User.GetId().String()).
+		RawJSON("Fileinfo", jsonFileInfo).
```

Comment:
We can't log the fileInfo. It might contain sensitive information. We should also NEVER log complete jsons (or structs, maps, ....) they can change later and then contain sensitive information.

Extract the useful information from the fileinfo if needed or remove the line completely